Objects/longobject.c

```diff
@@ -3790,6 +3860,43 @@ l_divmod(PyLongObject *v, PyLongObject *w,
     return 0;
 }
 
+/* Compute
+ *     *pmod = v % w
+ * An error should be raised if NULL is passed to pmod.
```

Comment:
And with respect to the later comment, if this is an internal function, and `pmod != NULL` is a precondition for calling it, then here you only need to say that `pmod` cannot be NULL. It's the function's responsibility to verify its preconditions are met (at least in a debug build).

Reply:
Fixed comment.